shapes/neurosciencegraph/datashapes/core/workflowengine/schema.json

```diff
@@ -0,0 +1,78 @@
+{
+  "@context": [
+    "https://incf.github.io/neuroshapes/contexts/schema.json",
+    {
+      "this": "https://neuroshapes.org/dash/workflowengine/shapes/"
+    }
+  ],
+  "@id": "https://neuroshapes.org/dash/workflowengine",
+  "@type": "nxv:Schema",
+  "imports": [
+    "https://neuroshapes.org/dash/softwareagent"
```

Comment:
I recommend to add an import for https://neuroshapes.org/commons/entity

Reply:
Sorry, forgot that one. Done.